libs/category/src/index.ts

```diff
@@ -17,14 +15,8 @@ export { DaffCategoryFacade } from './facades/category.facade';
 
 export { DaffCategoryModule } from './category.module';
 export { DaffCategoryMagentoDriverModule } from './drivers/magento/category-driver.module';
-export { DaffCategoryPageConfigTransformerInterface } from './drivers/interfaces/category-page-configuration-transformer.interface';
-export { DaffCategoryQueryManagerInterface } from './drivers/interfaces/category-query-manager.interface';
 export { DaffCategoryServiceInterface } from './drivers/interfaces/category-service.interface';
-export { DaffCategoryTransformerInterface } from './drivers/interfaces/category-transformer.interface';
 export { DaffCategoryDriver } from './drivers/injection-tokens/category-driver.token';
-export { DaffCategoryQueryManager } from './drivers/injection-tokens/category-query-manager.token';
-export { DaffCategoryResponseTransformer } from './drivers/injection-tokens/category-response-transformer.token';
-export { DaffCategoryTransformer } from './drivers/injection-tokens/category-transformer.token';
```

Comment:
I removed a lot of these because I'm going to export them out of a `**/magento/public_api.ts` file. can be ignored for now.